src/index.js

```diff
@@ -3,6 +3,19 @@ import ReactDOM from 'react-dom';
 import './index.css';
 import App from './App';
 import * as serviceWorker from './serviceWorker';
+import * as firebase from 'firebase/app';
```

Comment:
```suggestion
import * as firebase from 'firebase/app';
import 'firebase/auth';
```

Only with the second suggestion I created. I don't know how to apply suggestion for two different places in the code.